roles/kustomize_deploy/tasks/execute_step.yml

```diff
@@ -0,0 +1,189 @@
+---
+- name: Assert mandatory bits are defined
+  ansible.builtin.assert:
+    that:
+      - stage is defined
+      - stage_id is defined
+      - stage['path'] is defined
+      - stage.validations is defined
+      - stage.validations | length > 0
+      - stage['values'] is defined
+      - stage['values'] | length > 0
+
+- name: Group tasks under the same tag
+  tags:
+    - "deploy_va_stage_{{ stage_id }}"
+  block:
+    - name: Ensure source files exists
+      register: _src
+      when:
+        - item.src_file is not none
+        - item.src_file != 'none'
```

Comment:
Where's the "none" string comming from? Sonds like a weird value. Ok to `=! None` cause you check their against the python value, but I don't see where is this value coming from?

Reply:
Right thing to do:
- automation file gets rid of the "src_file" if not needed
- we check "is defined"